### DISCUSSION TITLE
JLCPCB 3D printing order issue

### DISCUSSION THREAD

**Discussion by RMNB** [srozum/film_camera_tester#53] [Q&A] [answered]
I got this email from JLCPCB after placing my order for parts. They're saying some of the walls are too thin specifically on the flexible hood (TPU), reflector (Resin), and sensor 6 face plate (resin). Did anyone get this warning and should i proceed at risk?

> As shown below, the wall thickness of those indicate red area are too thin, which has a high risk of loss, deformation ,crack, or damage during the printing process.
>Could you please confirm if the risks are acceptable for you?
>If not,for resin materials, could you please kindly increase it to be at least 0.8mm to proceed? 1.5mm will be better.Then we will activate the ' replace file' button for you.
>For TPU Plastic materials, could you please kindly increase it to be at least 1.2mm to proceed? 2.0mm will be better.Then we will activate the ' replace file' button for you.
>Besides ,due to the geometry of this workpiece, there is a high risk of deformation
>![image](https://github.com/user-attachments/assets/64e790ce-0d61-4334-91ca-7084c4104052)

**Comment by RobC62** [comment, marked as answer]
They will print fine. I have had two sets printed - all are perfect.
I had that message for many parts.

Edit: meant to add the parts they print via SLA are of almost injection moulding quality.

Also, check out the other discussion here where there are some modified sensor housing parts - the assembly and accuracy is much easier to achieve with them.

**Reply by RMNB** [reply]
Thank you!
